managed/services/management/backup/backups_service_test.go

```diff
@@ -377,7 +380,7 @@ func TestScheduledBackups(t *testing.T) {
 			tests.AssertGRPCErrorRE(t, codes.InvalidArgument, "PITR is only supported for logical backups", err)
 		})
 
-		t.Run("scheduling physical backups snapshot is successful", func(t *testing.T) {
+		t.Run("normal", func(t *testing.T) {
```

Comment:
It's embedded test, some parts of these information already present on higher level. Result test name will be  `TestScheduledBackups/mongo/normal`. `Physical` and `Snapshot` characteristics doesn't matter here, because we are not checking all possible options here and this test it the only happy path here.